demo/quiz.css

```diff
@@ -0,0 +1,14 @@
+ul {
+  list-style: none;
+}
+
+.Quiz-answer {
```

Comment:
OK! It's following https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md. I tend to use that in personal projects but I'm not really fussed about it.

I think we might end up removing all classes in the styling story and let the user of this component supply their own CSS classes. What do you say about leaving it to that story?